keystorage/aws/src/test/java/tech/pegasys/signers/aws/AwsSecretsManagerTest.java

```diff
@@ -386,27 +397,32 @@ void throwsAwayObjectsThatFailMapper() {
 
     assertThat(fetchedKeys).containsAll(expectedKeys);
     assertThat(fetchedKeys).doesNotContain(expectedKey);
+    assertThat(secretValueResult.getErrorCount()).isGreaterThan(0);
   }
 
```

Comment:
Be good to add a test case to cover when we can't retrieve any keys at all because can't connect to the secrets manager or the network is done etc. That is we fail early before we even go to map values.

Reply:
Added an AT to cover invalid credentials.